src/boot/AppEventHandlers.js

```diff
@@ -59,7 +59,10 @@ class AppEventHandlers extends PureComponent<Props> {
   handleNotificationOpen = (notification: Object) => {
     const { actions } = this.props;
     actions.saveInitialNotificationDetails(notification.getData());
-    setTimeout(() => handlePendingNotifications(notification), 600);
+    setTimeout(
```

Comment:
I know you've added this before, but why the 600 timeout. What happens if you call it right away?

Reply:
Basically, handleNotificationOpen is used when the app is in the background so to make sure everything is loaded.

Reply:
But are you **sure** 600 msecs is enough? Of course not. Every different device has different performance. Some are 5x slower than others. Using this, it will randomly not work sometimes.
Remove any timeouts and make sure it works without these.

Reply:
Ok, I'll see what can be done!